consensus/src/round_manager.rs

```diff
@@ -587,6 +587,22 @@ impl RoundManager {
             .author()
             .expect("Proposal should be verified having an author");
 
+        let payload_len = proposal.payload().map_or(0, |payload| payload.len());
+        let payload_size = proposal.payload().map_or(0, |payload| payload.size());
+        ensure!(
+            payload_len as u64 <= self.proposal_generator.max_block_txns(),
```

Comment:
This is not right. 

We should have different params - accept_max_block_txns and accept_max_block_bytes, and have them be 4-10x bigger than current values (like 10k and 3MB)

This way you are making it such that there cannot be different nodes with different values for this property in the network. If someone has a good network, and wants to increase their max_block_txns and max_block_bytes locally - they should be able to do so (within reason)

Also, even if we wanted the whole network to update these, with this code it would be impossible to gradually roll this change out. 
If we wanted to require everyone have same params for these - they should be on-chain, but I don't think we should require that.

Reply:
I'll update to 2x, 4x seems too much

Reply:
we can have explicit limits on both sender and receiver side if we want, but seems too much for now